docs/development/releasing.rst

```diff
@@ -257,18 +212,11 @@ clean-up tasks to finalize the process.
 Post-Release procedures
 -----------------------
 
-#. Go back to release branch (e.g., ``1.2.x``) and edit the ``version`` in
-   ``setup.cfg`` to be the next version number, but with
-   a ``.dev`` suffix at the end (e.g., ``1.2.3.dev``).  Then add and commit::
-
-      $ git checkout v1.2.x
-      <use your favorite editor on setup.cfg>
-      $ git add setup.cfg
-      $ git commit -m "Back to development: v<next_version>.dev"
-
-#. Also update the ``CHANGES.rst`` file with a new section for the next version.
+#. Go back to release branch (e.g., ``1.2.x``) and update the ``CHANGES.rst``
```

Comment:
Also need to tag the dev version here, e.g., v1.2.3.dev, for `setuptools_scm` to pick it up.

Reply:
This is already mentioned in the section 'Performing a Feature Freeze/Branching new Major Versions' - it is **not** needed inside release branches - that is, if you tag v1.2.2, setuptools_scm will automatically pick that up and call the version v1.2.3.dev in subsequent commits to the branch. The issue that requires adding .dev tags is *only* when doing branching from master.